### PR TITLE
ID - Addressed Issues with 4DVD Logo and Time Series Menu

### PR DESCRIPTION
**Pull request by zaydor**
I tried something a bit different to try to get the 4DVD logo showing up. And also addressed formatting issues with the Close button from the last pull request.

### CODE REVIEW

**Comment by dafrenchyman** [comment]
I'll get to this PR later in the week. Pretty busy right now.

File: src/app/timeseries-menu.component.html [hunk 1]
```diff
@@ -14,15 +14,26 @@ <h2 mat-dialog-title>Time Series</h2>
...
+    <p>Latitude: {{ this._model.settings.CurrGridBoxLat }}</p>
+    <p>Longitude: {{ this._model.settings.CurrGridBoxLon }}</p>
+    <br />
+    <br />
+    <button mat-raised-button (click)="closeTimeSeries()">
+      Close Time Series
+    </button>
   </mat-dialog-content>
+  <h2 mat-dialog-title style="text-align: center">
+    <u>Pressure Layers</u>
```
Comment (dafrenchyman) [inline src/app/timeseries-menu.component.html]: This one addition (_Pressure Layers_ text) seems to be messing up the size of the pop-up window and adding arrow bars to it (to scroll up and down the window).

File: src/app/timeseries-menu.component.html [hunk 2]
```diff
@@ -14,15 +14,26 @@ <h2 mat-dialog-title>Time Series</h2>
...
     <button mat-raised-button (click)="createCsvFromTimeseriesData()">
       Download Data
     </button>
+    <br />
+    <p>Latitude: {{ this._model.settings.CurrGridBoxLat }}</p>
+    <p>Longitude: {{ this._model.settings.CurrGridBoxLon }}</p>
+    <br />
+    <br />
+    <button mat-raised-button (click)="closeTimeSeries()">
+      Close Time Series
```
Comment (dafrenchyman) [inline src/app/timeseries-menu.component.html]: I don't like _Close Time Series_ button. I prefer how you put an X in the top right corner of the navbar to close that menu. Please do that here too, just add an X in the upper right corner of this window. I don't think we really need a close button, but that would look better.

File: src/app/timeseries-menu.component.html [hunk 3]
```diff
@@ -14,15 +14,26 @@ <h2 mat-dialog-title>Time Series</h2>
...
+      [yAxisLabel]="yValTitle()"
       [autoScale]="autoScale"
     >
     </ngx-charts-line-chart>
     <br />
     <button mat-raised-button (click)="createCsvFromTimeseriesData()">
       Download Data
     </button>
+    <br />
+    <p>Latitude: {{ this._model.settings.CurrGridBoxLat }}</p>
```
Comment (dafrenchyman) [inline src/app/timeseries-menu.component.html:34]: I like adding the Lat/Lon to this window. Nice job.

File: src/app/settings.ts [hunk 4]
```diff
@@ -193,6 +193,18 @@ export class Settings {
     return null;
   }
 
+  StringToArray(str): string {
```
Comment (dafrenchyman) [inline src/app/settings.ts]: I like the fact that you did this function. But the function/methods name should match what it's doing. It's not doing a `StringToArray`, it's actually generating the title.

**Review by dafrenchyman — CHANGES_REQUESTED** [review]
You are very close. Please address the comments mentioned in this PR.

**Comment by zaydor** [comment]
Addressed your comments, also moved the pressure layer checkboxes to the right of the graph as I thought it looked a bit cleaner than below the graph.

**Comment by dafrenchyman** [comment]
Opps,.. just noticed this. I'll get to it. Sorry.

**Review by dafrenchyman — CHANGES_REQUESTED** [review]
I just tested it and it still doesn't work.
When I check-mark more datasets to display it adds the arrow bars because the legend above it grows and grows as we add more datasets into it.

**Comment by zaydor** [comment]
Yeah, apologies. I am aware of that issue. It was working before without extending the popup box, but for whatever reason it turned into a single list. Working on a fix for it but was not able to test it while out of town. Will be fixed by Monday.

**Comment by dafrenchyman** [comment]
Hi Zaydor, 
I'm finally getting around to testing this, and the time series screen still doesn't work right.
I think we need to go about this a completely different way:
- You should split your work into different PRs. The other stuff you've done: The logo, the changes to removing `mrsharky_` would all have been easily approved if they were in their own PRs. When you write PRs they should be for changing/adding one thing, especially in this case when those would have been easy to approve.
- I do like what you've done with the time series, the only problem is that the window re-renders when it adds the legend in. I don't see the scroll lines anymore (which is good), but depending on the window size I have, I don't see all the buttons available. I think you have two options here: 
-- Remove the legend, and make our buttons the legend instead. Color code our legend buttons to match what they look like in the graph.
-- Keep the legend buttons on the bottom for now, but keep your lat/lon stuff. Eventually I think it would be beneficial to move to a different charting library (https://c3js.org/)

**Pull request closed without merging**